config/locales/en.yml

```diff
@@ -1627,11 +1627,11 @@ en:
               not_within_academic_cycle_description: The ITT start date is for a different academic year than the selected course. Enter a valid start date or <a class="govuk-link" href="%{url}">choose a different year</a>.
               hint_html: For example, 11 9 %{year}
             end_date:
-              blank: Enter an ITT end date
-              future: Enter an ITT end date closer to today
-              invalid: Enter a valid ITT end date
-              too_old: Enter a valid ITT end date
-              before_or_same_as_start_date: The ITT end date must be after the start date
+              blank: Enter an Expected end date
+              future: Enter an Expected end date closer to today
+              invalid: Enter a valid Expected end date
+              too_old: Enter a valid Expected end date
+              before_or_same_as_start_date: The Expected end date must be after the start date
```

Comment:
do we really want this capitalisation? seems off to me.

Reply:
definitely not, have updated